core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -32,6 +32,7 @@ import { WindowRef } from 'services/contextual/window-ref.service';
 @Component({
   selector: 'practice-tab',
   templateUrl: './practice-tab.component.html',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Could you please add a comment in the code to explain this? In general, it's good to address the "why" questions with comments since they wouldn't be obvious otherwise.

Reply:
Thanks @mahendra1290 for clarifying. Could you please add a code comment like Sean suggested above?